packages/app-project/src/helpers/fetchTranslations/fetchTranslations.spec.js

```diff
@@ -75,11 +70,22 @@ describe('helpers > fetchTranslations', function () {
     expect(translation?.strings?.content).to.equal('test project page')
   })
 
+  it('should handle language codes that have partial capitalization', async function () {
+    const translation = await fetchTranslations({
+      translated_id: 1234,
+      translated_type: 'project',
+      language: 'zh-CN',
+      fallback: 'en',
+      env: 'staging'
+    })
+    expect(translation?.strings?.display_name).to.equal('Chinese translation')
+  })
+
   it('should use the fallback language when translations don\'t exist', async function () {
     const translation = await fetchTranslations({
       translated_id: 1234,
       translated_type: 'project',
-      language: 'fr',
+      language: 'es',
```

Comment:
You’ll need to add a request matcher for this query too, or change the existing mock.